crates/ruff_linter/src/rules/pyflakes/snapshots/ruff_linter__rules__pyflakes__tests__F722_F722.py.snap

```diff
@@ -13,4 +13,68 @@ F722.py:13:4: F722 Syntax error in forward annotation: `List[int]☃`
    |
 13 | X: """List[int]"""'☃' = []
    |    ^^^^^^^^^^^^^^^^^^ F722
+14 |
+15 | # Type annotations with triple quotes can contain newlines and indentation
+   |
+
+F722.py:30:11: F722 Syntax error in forward annotation: `
+		int |
+str)
+`
```

Comment:
Unrelated to this PR, I think it would be useful to provide the syntax error that was raised during parsing this string content.